lang/en-gb.xml

```diff
@@ -506,11 +506,9 @@ After the download is complete, Vita3K will restart automatically and then insta
 		<line_sixth_part_one>A comprehensive guide on how to set-up Vita3K can be found on the</line_sixth_part_one>
 		<quickstart>Quickstart</quickstart>
 		<line_sixth_part_two>page.</line_sixth_part_two>
-		<line_seventh_part_one>Consult the Commercial game</line_seventh_part_one>
-		<compatibility_list>Compatibility List</compatibility_list>
-		<line_seventh_part_two>and the</line_seventh_part_two>
+		<line_seventh>Consult the Commercial game and the Homebrew compatibility list to see what currently runs.</line_seventh>
+		<commercial_compatibility_list>Commercial Compatibility List</commercial_compatibility_list>
```

Comment:
Called simply Compatibility List, not Commercial Game Compatibility List...

Reply:
not means for this, because overrun text